src/Microsoft.Health.Dicom.Core/Features/Update/UpdateOperationInstanceService.cs

```diff
@@ -4,14 +4,17 @@
 // -------------------------------------------------------------------------------------------------
 
 using System.Linq;
+using System.Net;
 using System.Threading;
 using System.Threading.Tasks;
 using EnsureThat;
+using FellowOakDicom;
 using Microsoft.Health.Dicom.Core.Exceptions;
 using Microsoft.Health.Dicom.Core.Features.Common;
 using Microsoft.Health.Dicom.Core.Features.Context;
 using Microsoft.Health.Dicom.Core.Features.Operations;
 using Microsoft.Health.Dicom.Core.Features.Routing;
+using Microsoft.Health.Dicom.Core.Messages.Update;
 using Microsoft.Health.Dicom.Core.Models.Operations;
 using Microsoft.Health.Dicom.Core.Models.Update;
 using Microsoft.Health.Operations;
```

Comment:
nit: I updated the class name to `UpdateInstanceOperationService`

Reply:
I updated the class and interface accordingly.